content/pages/health-care/about-va-health-care.md

```diff
@@ -7,6 +7,7 @@ concurrence: complete
 lastupdate: 2017-06-28
 collection: healthCare
 children: healthCareCoverage
+resource_id: HealthCare_About
```

Comment:
I user Proper_Case because I thought that would sell this best to the content editors if they choose to use this later on down the road.